src/main/java/com/palantir/docker/compose/connection/ContainerAccessor.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.docker.compose.connection;
+
+public interface ContainerAccessor {
```

Comment:
I'm indifferent to the method name. If this is supposed to be a common interface that multiple things implement `Cluster` might be a good name or `ContainerList`.